src/components/ContentPreview/Header.js

```diff
@@ -37,6 +43,8 @@ const Header = ({
     const closeMsg = intl.formatMessage(messages.close);
     const printMsg = intl.formatMessage(messages.print);
     const downloadMsg = intl.formatMessage(messages.download);
+    const fileId = file && file.id;
```

Comment:
Should this guard live in the OpenWith component itself?

Reply:
Good catch. I'll add a guard there to avoid unnecessary network calls, but this guard protects us from accessing a property on an undefined file prop since it's an optional prop.